lxd/network/driver_ovn.go

```diff
@@ -4236,6 +4236,7 @@ func (n *ovn) ForwardUpdate(listenAddress string, req api.NetworkForwardPut, cli
 		if err != nil {
 			return err
 		}
+
```

Comment:
We have a bit of an exception to that rule when considering reverters as is the case here.
We'll often keep the reverters in the same logical block as the code they revert.

Reply:
Agreed. Although this is now an automated linker check and it doesn't currently know about that, it also enforces newlines between each case block in a switch or select statement. I'll merge this for now as if we improve the linter then we'll need to undo some of what was done in https://github.com/lxc/lxd/pull/10617#discussion_r909921644

I merged that pr as its benefits appeared to outweigh these smaller conventions. Hopefully we can improve our linter and restore them in the future though.